bundles/org.openhab.binding.mffan/src/main/java/org/openhab/binding/mffan/internal/handler/MfFanHandler.java

```diff
@@ -72,22 +73,15 @@ public void initialize() {
         this.logger.debug("Initializing MfFan handler '{}'", getThing().getUID());
         updateStatus(ThingStatus.UNKNOWN);
         this.config = getConfigAs(MfFanConfiguration.class);
-        if (validateConfig(this.config)) {
-            try {
-                this.api = new FanRestApi(this.config.getIpAddress().trim(), this.httpClientFactory);
-                // handleCommand(RefreshType.REFRESH);
-            } catch (Exception e) {
-                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.COMMUNICATION_ERROR, e.getMessage());
-            }
-
+        if (MfFanConfiguration.validateConfig(this.config)) {
+            this.api = new FanRestApi(this.config.getIpAddress().trim(), this.httpClientFactory);
```

Comment:
Minor: The trim() should be part of the getIpAddress() as that method is responsible for returning a proper IpAddress